engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/FetchCachedResultsActor.scala

```diff
@@ -10,7 +11,7 @@ import scala.util.{Failure, Success}
 
 object FetchCachedResultsActor {
   def props(callCachingEntryId: CallCachingEntryId, replyTo: ActorRef, callCache: CallCache): Props =
-    Props(new FetchCachedResultsActor(callCachingEntryId, replyTo, callCache))
+    Props(new FetchCachedResultsActor(callCachingEntryId, replyTo, callCache)).withDispatcher(EngineDispatcher)
```

Comment:
I thought we had a special dispatcher for DB operations?

Looking at the list... was it the `IODispatcher`?

Also relevant for a lot of the other DB accessor actors in the PR